src/rantanplan/core.py

```diff
@@ -310,6 +376,7 @@ def get_scansion(text, rhyme_analysis=False):
             seen_tokens.append(token)
     if len(seen_tokens) > 0:
         lines.append({"tokens": get_syllables(seen_tokens)})
+    lines = get_metre(lines)
```

Comment:
I think this code can be much clearer if we reuse the existing structure. Something like:
```python
    if len(seen_tokens) > 0:
        metrical_pattern, prosodic_tokens = get_metre(seen_tokens)
        lines.append({
            "tokens": get_syllables(seen_tokens),
            "prosodic_tokens": prosodic_tokens,
            "metre": metrical_pattern,
        })
```
That would require to change the function `get_metre` slightly.

Reply:
Can we add it as an issue for future work on this?

Reply:
Can this be addressed now?

Reply:
It is, it's now:
```
    for line in lines:
        line.update(get_rhythmical_pattern(line, rhythm_format))
```